developer-docs-site/docs/reference/faq.md

```diff
@@ -0,0 +1,328 @@
+
+## Aptos Discord Community Roles
+
+### What are roles?
+
+A role gives a specific level of permission. As more and more contributors join Aptos Discord community, we created a few roles so that the community is better organized. 
+
+Check out the [Aptos Discord channel](https://discord.com/channels/945856774056083548/957486688710561872) `#roles` to get the role for yourself. 
+
+### What do these roles mean?
+
+- Node Operator: Any Aptos community member running Aptos nodes.
+- Developer: Any Aptos community member developing on Aptos Blockchain.
+- Aptonaut: Anyone who is exploring opportunities to contribute in the [Aptos Discord channel](https://discord.com/channels/945856774056083548/963777510473084988).
+
+### How do I get roles?
+
+Head over to the [Aptos Discord channel](https://discord.com/channels/945856774056083548/957486688710561872) `#roles` to get the role for yourself by reacting with the designated emoji. 
+
+## Aptos Token
+
+### Does Aptos have a token?
+
+There is no Aptos token. 
+
+### Is there an Aptos token for sale, presale or airdrop?
+
+There is no current token sale. Anyone claiming otherwise is running a scam and please make sure to report them in the [Aptos Discord channel](https://discord.com/channels/945856774056083548/951160191716585542) `#report-scams`. 
+
+Aptos Labs will post any updates and announcements in the [Aptos Discord channel](https://discord.com/channels/945856774056083548/946124073400938497) `#announcements`.
+
+## Working with Aptos Labs
+
+### I want to be a translator, what are my choices?
+
+Currently, any translation of Aptos Labs documentation from English to a non-English language is a community effort. Check the [Aptos Discord](https://discord.gg/zTDYBEud7U) channel.
+
+### Do you have an ambassador program?
+
+Yes. The Aptos community program will be announced next week. See Aptos Discord channel `#announcements`.
+TODO: Update with link to program.
+
+### Is Aptos is hiring?
+
+Yes! Check out this careers page on the Aptos Labs website: [aptoslabs.com/careers](https://aptoslabs.com/careers).
+
+### I have a commercial proposal, whom I should contact?
+
+Please contact `info@aptoslabs.com`. Note that Aptos Labs is not looking for investment. We would love to collaborate as a community!
+
+## Aptos devnet
+
+### What is the purpose of Aptos devnet?
+
+The Aptos devnet is a publicly deployed instance of the Aptos network, built for experimenting with new ideas. The Aptos devnet runs a set of validator test nodes. 
+
+While working with strategic partners and the developer community, we use Aptos devnet to learn where we can improve performance. We focus across all aspects of our community experience, from the Move developer and Move language to the Aptos blockchain network.
+
+### Do you have explorer for Aptos devnet?
+
+Yes. Click here: [https://explorer.devnet.aptos.dev](https://explorer.devnet.aptos.dev/)
```

Comment:
the url is irrelevant... they all point to the same explorer and the url doesn't change the network, we may want to flag a bug to Zihan.